src/components/cardbuilder/peoplecardbuilder.js

```diff
@@ -7,15 +7,17 @@
 import cardBuilder from './cardBuilder';
 
 export function buildPeopleCards(items, options) {
-    options = Object.assign(options || {}, {
+    options = {
+        ...options,
```

Comment:
I'm not sure why there was a default before, but we probably need to keep it to be safe. If `options` is undefined and we try to spread it, it will throw an exception. There are a few other places throughout that this change would need to be made as well.

```suggestion
        ...(options || {}),
```

Reply:
Good catch! For this instance at least, you could provide a default value for the parameter as an alternative too.

```ts
export function buildPeopleCards(items, options = {}) {
```

Reply:
Appreciate the suggestion. I believe I've added them all back.